game/views.py

```
@@ -2425,29 +2542,29 @@ def _classify_move(is_best, played_mv, best_mv, game_state):
     if is_best:
         return 'Best'
-        
+
     if not best_mv or not played_mv:
         return 'Mistake'
-        
+
     piece_vals = {'p': 1, 'n': 3, 'b': 3, 'r': 5, 'q': 9, 'k': 0}
-    
+
     def _simulate_and_evaluate(mv):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_simulate_and_evaluate`

(ANN202)